### ISSUE TITLE
Not able to install on device while using pods

### ISSUE THREAD

**Issue by himshikhar-sc** [tuist/tuist#4106]
Able to build the project but not able to run on device. Project includes swift packages, pods and few static frameworks as well.

Steps to reproduce:
1. `git clone https://github.com/himshikhar-sc/tuist-demo`
2. `tuist generate && pod install`
3. Run on device

If I do use `tuist generate` only, it runs on device.

Project: https://github.com/himshikhar-sc/tuist-demo

**Screenshot**

![](https://user-images.githubusercontent.com/96403797/150933840-a0d5dea7-e424-483d-b997-5764d0344105.png)

- OS: macOS
- Version 12.0
- Xcode version: 13.1

**Complete error**

```
Unable to install "Project_Name"
Domain: com.apple.dt.MobileDeviceErrorDomain
Code: -402620375
User Info: {
    IDERunOperationFailingWorker = IDEInstalliPhoneLauncher;
}
--
The code signature version is no longer supported.
Domain: com.apple.dt.MobileDeviceErrorDomain
Code: -402620375
User Info: {
    DVTRadarComponentKey = 261622;
    MobileDeviceErrorCode = "(0xE8008029)";
    "com.apple.dtdevicekit.stacktrace" = (
	0   DTDeviceKitBase                     0x000000011191aff8 DTDKCreateNSErrorFromAMDErrorCode + 272
	1   DTDeviceKitBase                     0x0000000111956550 __90-[DTDKMobileDeviceToken installApplicationBundleAtPath:withOptions:andError:withCallback:]_block_invoke + 160
	2   DVTFoundation                       0x0000000103af40e8 DVTInvokeWithStrongOwnership + 76
	3   DTDeviceKitBase                     0x00000001119562a0 -[DTDKMobileDeviceToken installApplicationBundleAtPath:withOptions:andError:withCallback:] + 1312
	4   IDEiOSSupportCore                   0x0000000121851be4 __118-[DVTiOSDevice(DVTiPhoneApplicationInstallation) processAppInstallSet:appUninstallSet:installOptions:completionBlock:]_block_invoke.292 + 2908
	5   DVTFoundation                       0x0000000103c1e248 __DVT_CALLING_CLIENT_BLOCK__ + 16
	6   DVTFoundation                       0x0000000103c1f834 __DVTDispatchAsync_block_invoke + 564
	7   libdispatch.dylib                   0x000000019d11cf60 _dispatch_call_block_and_release + 32
	8   libdispatch.dylib                   0x000000019d11ecac _dispatch_client_callout + 20
	9   libdispatch.dylib                   0x000000019d126430 _dispatch_lane_serial_drain + 672
	10  libdispatch.dylib                   0x000000019d126fa4 _dispatch_lane_invoke + 392
	11  libdispatch.dylib                   0x000000019d131808 _dispatch_workloop_worker_thread + 656
	12  libsystem_pthread.dylib             0x000000019d2d95b0 _pthread_wqthread + 288
	13  libsystem_pthread.dylib             0x000000019d2d82c4 start_wqthread + 8
);
}
--

Analytics Event: com.apple.dt.IDERunOperationWorkerFinished : {
    "device_model" = "iPhone12,1";
    "device_osBuild" = "15.2.1 (19C63)";
    "device_platform" = "com.apple.platform.iphoneos";
    "launchSession_schemeCommand" = Run;
    "launchSession_state" = 1;
    "launchSession_targetArch" = arm64;
    "operation_duration_ms" = 28905;
    "operation_errorCode" = "-402620375";
    "operation_errorDomain" = "com.apple.dt.MobileDeviceErrorDomain";
    "operation_errorWorker" = IDEInstalliPhoneLauncher;
    "operation_name" = IDEiPhoneRunOperationWorkerGroup;
    "param_consoleMode" = 0;
    "param_debugger_attachToExtensions" = 0;
    "param_debugger_attachToXPC" = 1;
    "param_debugger_type" = 5;
    "param_destination_isProxy" = 0;
    "param_destination_platform" = "com.apple.platform.iphoneos";
    "param_diag_MainThreadChecker_stopOnIssue" = 0;
    "param_diag_MallocStackLogging_enableDuringAttach" = 0;
    "param_diag_MallocStackLogging_enableForXPC" = 1;
    "param_diag_allowLocationSimulation" = 1;
    "param_diag_gpu_frameCapture_enable" = 0;
    "param_diag_gpu_shaderValidation_enable" = 0;
    "param_diag_gpu_validation_enable" = 0;
    "param_diag_memoryGraphOnResourceException" = 0;
    "param_diag_queueDebugging_enable" = 1;
    "param_diag_runtimeProfile_generate" = 0;
    "param_diag_sanitizer_asan_enable" = 0;
    "param_diag_sanitizer_tsan_enable" = 0;
    "param_diag_sanitizer_tsan_stopOnIssue" = 0;
    "param_diag_sanitizer_ubsan_stopOnIssue" = 0;
    "param_diag_showNonLocalizedStrings" = 0;
    "param_diag_viewDebugging_enabled" = 1;
    "param_diag_viewDebugging_insertDylibOnLaunch" = 1;
    "param_install_style" = 0;
    "param_launcher_UID" = 2;
    "param_launcher_allowDeviceSensorReplayData" = 0;
    "param_launcher_kind" = 0;
    "param_launcher_style" = 0;
    "param_launcher_substyle" = 0;
    "param_runnable_appExtensionHostRunMode" = 0;
    "param_runnable_productType" = "com.apple.product-type.application";
    "param_runnable_swiftVersion" = "5.5";
    "param_runnable_type" = 2;
    "param_testing_launchedForTesting" = 0;
    "param_testing_suppressSimulatorApp" = 0;
    "param_testing_usingCLI" = 0;
    "sdk_canonicalName" = "iphoneos15.0";
    "sdk_osVersion" = "15.0";
    "sdk_variant" = iphoneos;
}
```
--

System Information
macOS Version 12.0 (Build 21A344)
Xcode 13.0 (19234) (Build 13A233)

**Comment by danyf90** [comment]
If generate works, but after pod install it doesn't I guess it's a problem of your Podfile, not of tuist :D

Have you tried what is suggested [here](https://stackoverflow.com/questions/68467306/the-code-signature-version-is-no-longer-supported)?

**Comment by himshikhar-sc** [comment]
> If generate works, but after pod install it doesn't I guess it's a problem of your Podfile, not of tuist :D
> 
> Have you tried what is suggested [here](https://stackoverflow.com/questions/68467306/the-code-signature-version-is-no-longer-supported)?

Tried that, didn't work. Just thinking why this might happen because the same Podfile works when the project is not generated using Tuist.

**Comment by danyf90** [comment]
might it be because with the tuist generated project you are targeting iOS 15, while before you were using a lower min ios version?

**Comment by himshikhar-sc** [comment]
Should not be the case because I created this project just to share, so that issue can be replicated.

**Comment by himshikhar-sc** [comment]
The original issue I created was https://github.com/tuist/tuist/issues/4045 @luispadron @fortmarek

**Comment by danyf90** [comment]
> Should not be the case because I created this project just to share, so that issue can be replicated.

Not sure I got this :D how have you created that project? which ios version are you targeting?

**Comment by himshikhar-sc** [comment]
> > Should not be the case because I created this project just to share, so that issue can be replicated.
> 
> Not sure I got this :D how have you created that project? which ios version are you targeting?

1. Created a new Project using Xcode. Added the Podfile and did `pod install` and and was able to run on device. 
2. Created the Project.swift file. Did `tuist generate && pod install` .Tried running the same on device and got the above stated error.

Targeting iOS 15 in this project. This one is just a sample project that I created just to replicate the issue that I am getting in one another project.
